test/caching.bats

```diff
@@ -185,13 +185,19 @@ EOF
 }
 
 @test "can read previous version's cache" {
-    skip "old stacker will not build because libsquashfs1-dev is not installed"
-    git clone https://github.com/project-stacker/stacker
-    (cd stacker && make LXC_BRANCH=$(grep ^LXC_BRANCH Makefile | cut -d'=' -f 2) LXC_CLONE_URL=$LXC_CLONE_URL)
-
     # some additional testing that the cache can be read by older versions of
     # stacker (cache_test.go has the full test for the type, this just checks
     # the mechanics of filepaths and such)
+    local relurl="https://github.com/project-stacker/stacker/releases/download"
+    local oldver="v1.0.0-rc4"
```

Comment:
oldver=$(git ls-remote --refs --sort="version:refname" --tags $repo | cut -d/ -f3-|tail -n1)

should do it?

Reply:
oldver=$(git describe --abbrev=0 --tags)